### PR TITLE
chore: upgrade dependencies, internalize ui-components, remove unused assets

### PR DESCRIPTION
**Pull request by daniel-mader**
# Description of change
Please write a summary of your changes and why you made them.

## Links to any relevant issues
Be sure to reference any related issues by adding `fixes issue #`.

## How the change has been tested
Describe the tests that you ran to verify your changes.
Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Definition of Done checklist
Add an `x` to the boxes that are relevant to your changes.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: vite.config.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ export default defineConfig(async () => ({
     },
     fs: {
       allow: [
-        '/Users/daniel/.nvm/versions/node/v20.3.0/lib/node_modules/@impierce/ui-components/dist'
+        // '/Users/daniel/.nvm/versions/node/v20.3.0/lib/node_modules/@impierce/ui-components/dist'
```
Comment (nanderstabel) [inline vite.config.ts]: Can this be removed?

**Review by nanderstabel — DISMISSED** [review]
Great, thanks!

File: vite.config.ts (hunk 1 above)
Comment (daniel-mader) [reply vite.config.ts]: Sorry, my bad!

**Review by nanderstabel — APPROVED** [review]
Nice! :smile:

**Pull request merged**